README.md

```diff
@@ -14,7 +14,7 @@ A lightweight (no dependencies) crate for sending `systemd` service state notifi
 ## Quick start
 
 ```rust
-let _ = sd_notify::notify(true, &[NotifyState::Ready]);
+let _ = sd_notify::notify(false, &[NotifyState::Ready]);
```

Comment:
Can you also update this documentation for the changed API?

Reply:
Done.